gluster_brick_provisioning.adoc

```diff
@@ -0,0 +1,41 @@
+// vim: tw=79
+= Gluster brick provisioning
+
+This document describes some of the conventions followed when tendrl provisions
+bricks for gluster volumes. Also it provides guidelines that user has to follow
+if he creates brick manually and tendrl has to pick that.
```

Comment:
Do you mean that you would not be able to import already existing cluster with different naming scheme?